Src/VimWpf/Implementation/BlockCaret/BlockCaretController.cs

```diff
@@ -120,6 +120,8 @@ private void UpdateCaretDisplay()
                     kind = CaretDisplay.Invisible;
                     break;
                 case ModeKind.Insert:
+                    kind = CaretDisplay.Select;

```

Comment:
This will break users who have Visual Assist installed. They use `ITextCaret.IsHidden` to determine if they should / shouldn't be adding experiences like custom Intellisense. 

Maybe the fix here is to condition this behavior in some way and then disable it only for the times Visual Assist is installed. That information is already available in `IVisualAssistUtil.IsInstalled`. We could put something like `UseDefalutCaret` on `VimHost` and then have it return true only when Visual Assist is installed.

Reply:
Added this logic pretty much as you described.